packages/material-ui/src/Modal/Modal.js

```diff
@@ -310,6 +308,8 @@ class Modal extends React.Component {
         <div
           data-mui-test="Modal"
           ref={this.handleModalRef}
+          onKeyDown={this.handleDocumentKeyDown}
+          role="presentation"
```

Comment:
What's the motivation for using this role?

Reply:
@oliviertassinari Static HTML elements with event handlers require a role : https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md

Reply:
> Marking an element with the role presentation indicates to assistive technology that this element should be ignored; it exists to support the web application and is not meant for humans to interact with directly.

Reply:
It would be great to add it as a comment in the code. At least I would have enjoyed this information reading the source.

Reply:
Yes, it makes sense, added it together with the `event.stopPropagation()` to the `handleDocumentKeyDown` function like react-modal does.